CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Install and enable new development dependency:
+  [rubocop-performance](https://docs.rubocop.org/rubocop-performance/).
+
+### Changed
+- Update development dependencies.
+- Update RuboCop configuration, including new preferences, and refactor the
+  code to solve the new offenses.
+- Opt-in for rubygems [MFA requirement](https://guides.rubygems.org/mfa-requirement-opt-in/).
+- Replace Travis CI with a GitHub Actions CI Workflow.
```

Comment:
Should we even document this? All of these are internal changes which do not affect users of the gem in any way.

Removing this would help to reduce other people's cognitive load when they read the changelog because they want to upgrade the gem in their projects.

Reply:
I agree and think a simple "Internal CI workflow updates." entry in the changelog should be enough in this case.

Reply:
You know what? I think we can leave this as is. We can always change it before making an official release.